pages/docs/tutorials/javascript/setting-up.md

```diff
@@ -1,106 +1,151 @@
+   
+   ```kotlin
+   styledDiv {
+               css {
+                   +WelcomeStyles.textContainer
+               }
+               +"Hello ${state.name}!"
+               +" Your name backwards is ${state.name.reversed()}!"
+           }
+   ```
+   
+   </div>
+   
+3. Save changes to the file.
+
+4. Go to the browser and enjoy the result.
+   
+<img class="img-responsive" src="{{ url_for('tutorial_img', filename='javascript/setting-up/js-output-2.png') }}" alt="Web browser with a reversed name" />
+
+### Add an image
+
+1. Open the file `welcome.kt` in **src** \| **main** \| **kotlin**.  
+
+2. Add the `div` element with an image `img`.  
+   
+   > Make sure that you import these packages - `react.dom.*` and `styled.*`.
+   {:.note}       
+   
+   <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+   
+   ```kotlin
+   div {
+           img(src = "https://placekitten.com/408/287") {}
+       }
+   ```
+   
+   </div>                                                                                                                                                                                                                                                        >
+   
+3. Save changes to the file.
+
+4. Go to the browser and enjoy the result.
+   
+<img class="img-responsive" src="{{ url_for('tutorial_img', filename='javascript/setting-up/js-output-3.png') }}" alt="Web page with with an image" width="500"/>
+
+### Add a button that changes text
+
+1. Open the file `welcome.kt` in **src** \| **main** \| **kotlin**.  
+
+2. Add the `button` element with the `onClickFunction` event.  
+   
+   > Make sure that you import the package `kotlinx.html.js.*`.
+   {:.note}         
+   
+   <div class="sample" markdown="1" theme="idea" mode="kotlin" data-highlight-only>
+   
+   ```kotlin
+    button {
+        attrs.onClickFunction = {
+            setState(
+                    WelcomeState(name = "Some name")
+            )
+        }
+        +"Change name"
+    }   
+   ```
+   
+   </div>                                                                                                                                                                                                                                                              >
+   
+3. Save changes to the file.
 
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
-```kotlin
-plugins {
-    kotlin("multiplatform") version "1.3.70"
-}
+4. Go to the browser and enjoy the result.
+   
+<img class="img-responsive" src="{{ url_for('tutorial_img', filename='javascript/setting-up/js-output-4.png') }}" alt="Web page with a button" width="500"/>
 
-group = "org.example"
-version = "1.0-SNAPSHOT"
+## What's next?
 
-repositories {
-    mavenCentral()
-}
+Once you have created your first application, you can go to Kotlin hands-on labs and pass long-form tutorials on Kotlin/JS.
```

Comment:
I think "complete" would be better than "pass". The latter sounds so much like an exam :)

Reply:
✅